app.py

```diff
@@ -143,9 +150,211 @@ def get_accounts():
+                                    "iban": "LV68LATB000222PSD2002",
+                                    "currency": "EUR",
+                                    "product": "Fremdwährungskonto",
+                                    "cashAccountType": "CurrentAccount",
+                                    "name": "US Dollar Account",
+                                    "_links": {
+                                        "balances": {
+                                            "href": "/v1/accounts/3dc3d5b3-7023-4848-9853-f5400a64e81g/balances"},
+                                        "transactions": {
+                                            "href": "/v1/accounts/3dc3d5b3-7023-4848-9853-f5400a64e81g/transactions"}}
+                                },
+                            ]
+                    }),
+                status=200,
+                headers=headers
+            )
+        else:
+            return Response(
+                json.dumps(
+                    "No account with such iban was found"
+                ),
+                status=400
+            )
+
+    return err_resp
+
+
+@app.route(api_path + "/v1/funds-confirmations", methods=["POST"])
+@app.route(api_path + "/v1/funds-confirmations/", methods=["POST"])
+def get_funds_confirmations():
+    header = request.headers
+    if header.get("X-Request-ID") == current_req_id and header.get("Consent-ID") == consent:
+        headers["X-Request-ID"] = current_req_id
+        headers["Content-Type"] = "application/json"
+        access = json.loads(request.data).get("access")
+        req_iban = access["iban"]
+        instructed_amount = json.loads(request.data).get("instructedAmount")
+        req_amount = instructed_amount["amount"]
+        req_currency = instructed_amount["currency"]
+        with open("./res/balances.json") as f:
+            balances = json.load(f)
+            if req_iban not in balances.keys():
+                return err_resp
+            cur_amount = balances[req_iban][req_currency]
+        if req_amount > cur_amount:
+            headers["X-Request-ID"] = current_req_id
+            return Response(
+                json.dumps(
+                    {
+                        "fundsAvailable": "false"
+                    }),
+                status=200,
+                headers=headers
+            )
+        else:
+            headers["X-Request-ID"] = current_req_id
+            return Response(
+                json.dumps(
+                    {
+                        "fundsAvailable": "true"
+                    }),
+                status=200,
+                headers=headers
+            )
+    return err_resp
+
+
+@app.route(api_path + "/v1/payments/sepa-credit-transfers", methods=["POST"])
+@app.route(api_path + "/v1/payments/sepa-credit-transfers/", methods=["POST"])
+def transfer_sepa_credit():
+    header = request.headers
+    if header.get("X-Request-ID") == current_req_id and header.get("Consent-ID") == consent \
```

Comment:
How does this if statement checks that the TPP who send this request has consent to the debtorAccount and the creditorAccount.

Reply:
no